lib/kumascript/server.js

```diff
@@ -315,6 +315,11 @@ var Server = ks_utils.Class({
                 env.revalidate_at = (new Date()).getTime();
             }
 
+            // Add a clone of the interactive-examples settings to "env".
+            env.interactive_examples = _.clone(
```

Comment:
This should be:

```
env.interactive_examples = ks_conf.nconf.get('interactive_examples');
```

Reply:
Oh man, good catch, sorry about that! Fix coming soon...